health/REFERENCE.md

```diff
@@ -745,21 +743,21 @@ Netdata will create alarms for all dimensions of the chart.
 
 ## Troubleshooting
 
-You can compile Netdata with [debugging](../daemon/README.md#debugging) and then set in `netdata.conf`:
+You can compile Netdata with [debugging](/daemon/README.md#debugging) and then set in `netdata.conf`:
 
 ```yaml
 [global]
    debug flags = 0x0000000000800000
 ```
 
-Then check your `/var/log/netdata/debug.log`. It will show you how it works.
-Important: this will generate a lot of output in debug.log.
+Then check your `/var/log/netdata/debug.log`. It will show you how it works. Important: this will generate a lot of
+output in debug.log.
 
-You can find the context of charts by looking up the chart in either
-`http://your.netdata:19999/netdata.conf` or `http://your.netdata:19999/api/v1/charts`.
+You can find the context of charts by looking up the chart in either `http://NODE:19999/netdata.conf` or
+`http://NODE:19999/api/v1/charts`, replacing `NODE` with the IP address or domain name for your Agent dashboard.
```

Comment:
the same suggestion given above can be used here, please, use `hostname` instead `domain name`.